build_manylinux_arm64.sh

```diff
@@ -19,7 +19,8 @@ cmake ${SAMNTDIR}/api -DCMAKE_BUILD_TYPE=Release -DSAMAPI_EXPORT=1 -DSAM_SKIP_AU
 make -j6
 
 cd $PYSAMDIR
-for PYTHONENV in cp38-cp38 cp39-cp39 cp310-cp310
+#for PYTHONENV in cp38-cp38 cp39-cp39 cp310-cp310 cp311-cp311
+for PYTHONENV in cp311-cp311
```

Comment:
Please fix this

Reply:
> Please fix this

Fixed - I had to build 3.11 separately - was failing when built with 3.8, 3.9 and 3.10 - I think it was caused by intermittent outage of the Wind Toolkit.